pages/data-providers/[data-provider-id]/compliance.jsx

```diff
@@ -40,26 +40,31 @@ const SidebarContent = ({ context: { oai, originalId, authors, title } }) => {
  *       of Layout.Main and explicitly pass props.
  *       We should get rid out of it!
  */
-const DepositDates = ({
-  className,
-  store,
-  tag: Tag = 'main',
-  ...restProps
-}) => (
+const Compliance = ({ className, store, tag: Tag = 'main', ...restProps }) => (
   <Tag className={[styles.container, className].join(' ')} {...restProps}>
-    <h1>Deposit compliance</h1>
+    <h1>REF 2021 Compliance</h1>
 
-    <Card className={styles.complianceLevel} tag="section">
+    <Card className={styles.dataOverview} tag="section">
       <h2>{texts.compliance.title}</h2>
```

Comment:
If you change the scope it would be good to change it everywhere. In `texts` case may be better to create a new one.